bib/cmd/bootc-image-builder/main.go

```diff
@@ -176,11 +176,15 @@ func saveManifest(ms manifest.OSBuildManifest, fpath string) error {
 	if err != nil {
 		return fmt.Errorf("failed to create output file %q: %s", fpath, err.Error())
 	}
-	// nolint:errcheck
-	defer fp.Close()
+	defer func() {
```

Comment:
if we use this I think `errors.Join(fp.Close())` is fine (the path is already part of path error, see ,https://github.com/golang/go/blob/master/src/os/file_unix.go#L316) I slightly prefer to just have a final `return fp.Close()` here but no strong opinion